test/relations.test.js

```diff
@@ -3629,6 +4828,7 @@ describe('relations', function () {
           });
           cb(null, summary);
         });
+        return cb.promise;
       };
 
       summarize.shared = true; // remoting
```

Comment:
With bluebird you can do that using .spread(). I'm not sure if that is something standard across promise libraries or weather it's just bluebird specific but it's certainly nice to enable you to pass data between multiple promise scopes.